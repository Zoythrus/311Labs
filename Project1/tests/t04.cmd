 text4_1.txt text4_2.txt 
