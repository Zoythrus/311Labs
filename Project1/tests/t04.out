Life is not about finding yourself. Life is about creating yourself. George Bernard Shaw. Progress is impossible without change, and those who cannot change their minds cannot change anything. George Bernard Shaw. The first step toward success is taken when you refuse to be a captive of the environment in which you first find yourself. Mark Caine. It had long since come to my attention that people of accomplishment rarely sat back and let things happen to them. They went out and happened to things. Leonardo Da Vinci. The good news is that the moment you decide that what you know is more important than what you have been taught to believe, you will have shifted gears in your quest for abundance. Success comes from within, not from without. Elie Wiesel. It is impossible to escape the impression that people commonly use false standards of measurement â€” that they seek power, success and wealth for themselves and admire them in others, and that they underestimate what is of true value in life. Sigmund Freud. 
